R/calculate-effect-sizes.R

```diff
@@ -424,6 +454,43 @@ batch_calc_ES <- function(dat,
       ES_weights <- 
         ES_ests_long %>%
         dplyr::mutate(weights = 1L)
+    } else if (weighting %in% c("nA", "n_A")) {
+      
+      nA_weights <- 
+        n_weights %>% 
+        dplyr::select(-c(nB, nAnB, sum_inv_nAnB))
+      
+      ES_weights <- 
+        ES_ests_long %>% 
+        dplyr::left_join(nA_weights) %>% 

```

Comment:
Is there a way to suppress the message that displays when `left_join`ing without specifying a `by =` argument?